src/signalhandler.cc

```diff
@@ -363,11 +392,15 @@ void InstallFailureSignalHandler() {
   for (size_t i = 0; i < ARRAYSIZE(kFailureSignals); ++i) {
     CHECK_ERR(sigaction(kFailureSignals[i].number, &sig_action, NULL));
   }
+#elif defined(OS_WINDOWS)
+  typedef void (*SignalHandlerPointer)(int); 
+  SignalHandlerPointer previousHandler;  
+  previousHandler = signal(SIGABRT, &FailureSignalHandler); 
```

Comment:
This is where Windows gets weird. While `SIGNAL.H` defines all of these signals as valid parameters to `signal()`, it does so only for compatibility. Explicitly, the signals `SIGILL` and `SIGTERM` are not generated on Windows (though you can explicitly raise them).

However, I think that since you can explicitly raise these signals (and it appears that `SIGFPE` and `SIGSEGV` might be raised on Windows), I'll change this to handle them all.